database/migrations/2020_12_10_103112_create_messages_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMessagesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('messages', function (Blueprint $table) {
+            $table->id();
+            $table->bigInteger( 'from');
+            $table->bigInteger( 'to');
+            $table->text( 'message');
+            $table->tinyInteger('is_read');
+            $table->timestamps();
```

Comment:
I am missing the foreign key constraints here.. There is no database relationship between the user and messages